CHANGELOG.md

```diff
@@ -19,12 +19,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 <!-- ### Security -->
 
-## [1.15.0] - 
+## [1.15.0] - 2020-10-22
 
 ### Added
 
 - Script for fixing invalid backup files, as a workaround to issue [#513] [#548](https://github.com/openkfw/TruBudget/pull/548)
 
+### Fixed
+
+- A group can now only be edited by the user who created it, other users don't see the edit button [#556](https://github.com/openkfw/TruBudget/issues/556)
```

Comment:
`A group can now only be edited by the user who created it` - that's the same behaviour as before